code/objects/items/equipable.dm

```diff
@@ -1,2 +1,32 @@
-/obj/item/equipable
-	var/item_slot
\ No newline at end of file
+#define HEAD_LAYER FLOAT_LAYER - 0.1
+#define BELT_LAYER FLOAT_LAYER - 0.2
+#define ARMOR_LAYER FLOAT_LAYER - 0.3
+#define	CLOTHING_LAYER FLOAT_LAYER - 0.4
+#define LEGS_LAYER FLOAT_LAYER - 0.5
+
+/obj/item/equipable
+	var/item_slot
+	var/worn_icon
+	var/worn_icon_state
+	var/cursed = FALSE
+	var/stats = list()

```

Comment:
I am unsure if equipment should directly affect stats. I think stats should be modified by modifiers on the items rather than the item itself.
Thought I guess stats can be pre-calculated from modifiers and be stored here.